components/authz-service/server/v2/project.go

```diff
@@ -317,15 +330,54 @@ func (s *ProjectState) ListProjectsForIntrospection(
 
 func (s *ProjectState) DeleteProject(ctx context.Context,
 	req *api.DeleteProjectReq) (*api.DeleteProjectResp, error) {
-	err := s.store.DeleteProject(ctx, req.Id)
-	switch err {
-	case nil:
-		return &api.DeleteProjectResp{}, nil
-	case storage_errors.ErrNotFound:
-		return nil, status.Errorf(codes.NotFound, "no project with ID %q found", req.Id)
-	default: // some other error
+
+	s.log.Info("server purge: START")
```

Comment:
can we remove this log statement or add more information to it (like what we're trying to delete)